TL improvement suggestion (#366)
07th-mod/higurashi-console-arcs: Update/omot_017.txt

				NULL, " After a few days of sightseeing in Hinamizawa, my wife and I are planning to go to a private hot spring in Wakura.", Line_WaitForInput);
		ModPlayVoiceLS(4, 13, "ps2/13/211300119", 540, TRUE);
		OutputLine(NULL, "————と、いうことにしておいてくれ」",

> 				NULL, " ...I want to keep it that way.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " ...At least, that's the story.\"", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "公安のエリ一ト街道を突き進み、忙しいどころではないほど過密な毎日を送っている赤坂が地方に起こった事件……、",
			   NULL, "I'm sure Akasaka, who was leading a very busy life as an elite member of Public Security, had no time to deal with an incident in a remote town...", Line_Normal);
		ClearMessage();

		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "にすらまだなっていない疑惑に手を伸ばしている余裕はまったくないはずだし、それに見逃したところで、誰からの非難も受けるものではないだろう。",

> 			   NULL, "or more precisely these allegations about there. So there'd be no shame in ignoring them.", Line_Normal);

Reviewer: ```suggestion
			   NULL, "And there was no guarantee these allegations would even come true, so there'd be no shame in ignoring them.", Line_Normal);
```

//r…そして一通り話し終わった後、赤坂が背後のドアにちら、と目を向ける。…結構、時間も過ぎてしまった。入江と詩音も、いつこちらの様子をうかがいに来てもおかしくないだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そして一通り話し終わった後、赤坂が背後のドアにちら、と目を向ける。…結構、時間も過ぎてしまった。入江と詩音も、いつこちらの様子をうかがいに来てもおかしくないだろう。",

> 			NULL, "And after finishing the explanation, Akasaka glanced back at the door behind him. ...Quite a lot of time had passed. I'll be not surprised if Irie and Shion walk in on this right now.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And after finishing the explanation, Akasaka glanced back at the door behind him. ...Quite a lot of time had passed. I wouldn't be surprised if Irie and Shion walked in on this right now.", Line_Normal);
```

			NULL, "\"So... I guess you heard all that from Doctor Irie at this clinic.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300160", 256, TRUE);
	OutputLine(NULL, "雛見沢症候群の研究の規模と資金から見ても、鷹野はトカゲの尻尾だ」",

> 			NULL, " In terms of the scale and funding into Hinamizawa Syndrome research, Takano is just one of the underlings who can be left holding the bag.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " In terms of the scale and funding into Hinamizawa Syndrome research, Takano is just the tail of the lizard.\"", Line_Normal);
```

	ClearMessage();

//赤坂rvS14/13/211300161.「『東京』という組織が何をたくらんでいるのかを突き止めない以上、根元を調べて絶たなければ、別のやつが来て梨花ちゃんを消そうと考えるかもしれない」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300161", 256, TRUE);
	OutputLine(NULL, "「『東京』という組織が何をたくらんでいるのかを突き止めない以上、根元を調べて絶たなければ、別のやつが来て梨花ちゃんを消そうと考えるかもしれない」",

> 			NULL, "\"Unless we can figure out what this \"Tokyo\" organization is really planning and collapse the base of it, another person could come along and try to eliminate you, Rika-chan.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Unless we can figure out what this \"Tokyo\" organization is really planning and take it down from inside, another person could come along and try to eliminate you, Rika-chan.\"", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 13, "ps2/13/211300169", 540, TRUE);
		OutputLine(NULL, "雪絵は実家で娘の世話で、身辺にも警護を入れているから問題ない」",

> 				NULL, " Yukie is now at my parents' house to take care of our daughter, and they have personal security in place so it won't be a problem.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Yukie is now at my parents' house taking care of our daughter, and they have personal security in place so it won't be a problem.\"", Line_Normal);
```

//r明日は村を総出で、鉄平を捜索するらしい。…警察に引き渡す前に、せめて目にもの見せなければ気がすまない、ということだろうか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "明日は村を総出で、鉄平を捜索するらしい。…警察に引き渡す前に、せめて目にもの見せなければ気がすまない、ということだろうか。",

> 			NULL, "Apparently, all of them are going to search this village for Teppei tomorrow. I guess it's because they have to get even with him before handing him over to the police.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Apparently, all of them are going to search this village for Teppei tomorrow. I guess it's because they want to get even with him before handing him over to the police.", Line_Normal);
```

//rおそらく彼の頭の中では、あらゆる事態に対応すべく今後の調査と、人員の配置について考えをめぐらせているのだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おそらく彼の頭の中では、あらゆる事態に対応すべく今後の調査と、人員の配置について考えをめぐらせているのだろう。",

> 			NULL, "In his head, he's probably thinking about ways to allocate staff and about future investigations to address this situation.", Line_Normal);

Reviewer: ```suggestion
			NULL, "In his head, he's probably thinking about ways to allocate staff, and about future investigations to address this situation.", Line_Normal);
```

//r富竹としても協力したいが、上への説得には時間がかかる…と、その都度申し訳なさそうに返していたことを思い出す。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "富竹としても協力したいが、上への説得には時間がかかる…と、その都度申し訳なさそうに返していたことを思い出す。",
			NULL, "\"I do want to properly cooperate with you, but it would take far too much time to persuade the upper ranks...\" I remember Tomitake looked sorry every time he told that to Irie.", Line_Normal);
	ClearMessage();

//rその、申し訳ないと思う気持ち。…その引け目は、相手の裏切りの疑惑をきっかけに反感へと転嫁しやすい。今の富竹の心境は、まさにそれだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その、申し訳ないと思う気持ち。…その引け目は、相手の裏切りの疑惑をきっかけに反感へと転嫁しやすい。今の富竹の心境は、まさにそれだろう。",

> 			NULL, "That was caused by a feeling of guilt, ...which is so easy to pass on animosity by making someone suspect betrayal. And that's what he is feeling right now.", Line_Normal);

Reviewer: ```suggestion
			NULL, "That was caused by guilty feelings. ...It's is so easy to pass on animosity by making someone suspect betrayal. And that's what he is feeling right now.", Line_Normal);
```
